docs/deployment/pull-backup.rst

```diff
@@ -0,0 +1,92 @@
+.. include:: ../global.rst.inc
+.. highlight:: none
+
+Backing up in pull mode
+=======================
+
+Assume you have a pull backup system set up with borg, where a backup server
+pulls the data from the target via sshfs.
+
+In this mode, the backup client's file system is mounted via sshfs on the backup
+server. Pull mode is even possible if the SSH connection must be established by
+the client via remote tunnel.
+
+Creating backup
+---------------
+
+In this approach the client file system is simply mounted and then backed up.
+
+::
+
+    mkdir /mnt/sshfs
+    sshfs root@TARGET_IP:/ /mnt/sshfs
+    borg create /REPOSITORY::archive /mnt/sshfs
+
+Note that this will back up every file with the ``/mnt/sshfs`` prefix. You can
+get rid of this prefix by changing the working directory before creating the
+backup:
+
+::
+
+    cd /mnt/sshfs
+    borg create /REPOSITORY::archive
```

Comment:
Why not just document the latter way if it is more appropriate?

But if you cd into there, you also need to cd out of there again.

Reply:
Also, there should be a note here, that if you backup server has root access to your clients, you need to have a lot of trust in the backup server (which is quite the opposite of borg's usualy threat model where the backup server is relatively untrusted) and a compromise of it would be a quite "global" desaster on potentially all the clients it has root access to.

Reply:
Also, as everything goes over sshfs, borg can only backup stuff supported by sshfs, which might be less than when directly access a local filesystem.

Check: filenames (encoding?), xattrs, ACLs, bsdflags, ...

Reply:
Also, if there is a UID/username GID/groupname mapping issue, it should be already mentioned here.

Reply:
Good points. I'll add a section discussing unsupported features when using sshfs and the security aspects.